recipes/evo/recipe.yaml

```diff
@@ -0,0 +1,66 @@
+context:
+  version: 1.31.1
+  python_min: 3.10
```

Comment:
```suggestion
context:
  name: evo
  version: 1.31.1
  python_min: 3.10
```

Reply:
Thanks, just noticed that in parallel :)